fixtures/eurolite/led-bar-12-qcl-rgba-bar.json

```diff
@@ -0,0 +1,387 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Bar-12 QCL RGBA Bar",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Rainer Jung"],
+    "createDate": "2022-05-21",
+    "lastModifyDate": "2022-05-21"
+  },
+  "links": {
+    "manual": [
+      "https://www.steinigke.de/download/51930396-Manual-120587-1.200-eurolite-led-bar-12-qcl-rgba-bar-de_en.pdf"
+    ],
+    "productPage": [
+      "https://www.steinigke.de/en/mpn51930396-eurolite-led-bar-12-qcl-rgba-bar.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=ZIj0aZyng1Q"
+    ]
+  },
+  "physical": {
+    "dimensions": [1090, 65, 100],
+    "weight": 2.9,
+    "power": 24,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [17, 17]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "defaultValue": 0,
+      "highlightValue": 255,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "defaultValue": 0,
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": 0,
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": 0,
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Amber": {
+      "defaultValue": 0,
+      "highlightValue": 255,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "Auto Program": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [10, 119],
+          "type": "Effect",
+          "effectPreset": "ColorFade",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [120, 255],
+          "type": "Effect",
+          "effectPreset": "ColorJump",
+          "soundControlled": true,
+          "soundSensitivityStart": "low",
+          "soundSensitivityEnd": "high"
+        }
```

Comment:
The manual only lists "Auto mode" and "Sound mode". I guess that you checked that the former is color fading and the latter is color jumping?

Reply:
The color is fading for the first option.  
For the second option. it's fading, but pretty quickly, I would estimate around ¼s, so I went with Jump.
Is there a best practice what to choose here?

Reply:
Also, the range 10-119 does not make a difference.

Reply:
Okay, I think both `ColorJump` and `ColorFade` would be fine then.

Is the sound sensitivity controllable then? I.e. is DMX value 120 different from DMX value 255?

Reply:
I'd still stick with `ColorFade`, as it's a fade, even if short and not controllable.   
I couldn't see any difference between different values on the sound mode (120-255).

Reply:
Oh, I didn't see that you actually already removed `soundSensitivityStart`/`soundSensitivityEnd` in [`2fc703f`](https://github.com/OpenLightingProject/open-fixture-library/pull/2624/commits/2fc703f3c71fb68de178c81d84a9c8e32d35e8cd). Then this is fine now :)